misk-admin/src/main/kotlin/misk/web/metadata/jvm/JvmMetadataAction.kt

```diff
@@ -0,0 +1,91 @@
+package misk.web.metadata.jvm
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import misk.moshi.adapter
+import misk.web.Get
+import misk.web.RequestContentType
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.dashboard.AdminDashboardAccess
+import misk.web.mediatype.MediaTypes
+import java.lang.management.RuntimeMXBean
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Conveys information about the current JVM
+ */
+@Singleton
+class JvmMetadataAction : WebAction {
+
+  private val runtimeMxBean: RuntimeMXBean
+  private val moshiAdapter: JsonAdapter<JvmRuntimeResponse>
+
+  @Inject constructor(runtimeMxBean: RuntimeMXBean, moshi: Moshi) {
+    this.runtimeMxBean = runtimeMxBean
+    // Indent to make the output more readable since the response can be quite large!
+    this.moshiAdapter = moshi.adapter<JvmRuntimeResponse>().indent("  ")
```

Comment:
in other frameworks I've seen affordance to configure a different serializer for things like admin consoles and the like (where it makes sense to have pretty output turned on generally). I didn't see any such affordance in misk so just doing this locally here.